Formula/tarantool.rb

```diff
@@ -22,7 +22,8 @@ class Tarantool < Formula
   uses_from_macos "ncurses"
 
   def install
-    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    sdk = MacOS.sdk_path_if_needed
```

Comment:
```suggestion
    sdk = MacOS.sdk_path_if_needed if MacOS.version >= :big_sur
```
It looks like you actually only need this for Big Sur. It'll be `nil` for earlier versions, and be interpolated as an empty string.